napalm_junos/junos.py

```diff
@@ -55,13 +55,14 @@
 class JunOSDriver(NetworkDriver):
     """JunOSDriver class - inherits NetworkDriver from napalm_base."""
 
-    def __init__(self, hostname, username, password, timeout=60, optional_args=None):
+    def __init__(self, hostname, username, password='', timeout=60, optional_args={}):
```

Comment:
Need to switch this back to optional_args=None (as otherwise we will have a problem with Python mutable).